checkly/checkly-cli
# feat: github actions smoke-test

I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## ⚙️ Affected Components

- [ ] Commands
- [ ] Modules



### 📝 Notes for the Reviewer

Add GH Action smoke-test which executes the following:
1. project init
2. deploying project (create project, create check)
3. update check
4. delete check
5. delete project




### 🏗️ New Dependency Submission



### 🎟 Affected Issue
